test/fixtures/3100/1.4.0S01.telnet

```diff
@@ -0,0 +1,18 @@
+----------------------------------------------------------------------------
+  Model: UC-3121-T-US-LX
+  Boot Loader Version: 1.4.0S01         CPU TYPE: 1000MHz

```

Comment:
This is a standard bootloader. Do you want to add the custom one as well?

Reply:
https://github.com/imZack/moxa-image-uploader/pull/14/files#diff-be0e34c4176429dddb7646b2b2e8e019 yep, it's there.